docs/src/modules/utils/getProductInfoFromUrl.ts

```diff
@@ -10,9 +10,11 @@ export type MuiProductId =
   | 'docs'
   | 'x-data-grid'
   | 'x-date-pickers'
-  | 'x-charts';
+  | 'x-charts'
```

Comment:
```suggestion
  | 'x-charts'
  | 'x-tree-view'
```

Hijacking this PR for @flaviendelangle